README.rst

```diff
@@ -49,16 +49,23 @@ The -L flag can be used to whitelist certain words that are comma-separated plac
 
     codespell -S, --skip=
 
-Comma-separated list of files to skip. It accepts globs as well.  Examples:
+The -S (--skip) flag is used to skip over a comma-separated list of files and/or directories. It accepts globs as well. **Important notes:** Do not append a '/' to a directory you want to skip. This renders the entry for that particular directory ineffective. Also 
 
-* to skip .eps & .txt files, invoke ``codespell --skip="*.eps,*.txt"``
+Examples:
 
-* to skip directories, invoke ``codespell --skip="./src/3rd-Party,./src/Test"`` ::
+* Skip .eps & .txt files, invoke one of::
 
+    codespell --skip="*.eps,*.txt"
+    codespell -S *.eps,*.txt
+
+* Skip directories, invoke one of::
+
+      codespell --skip="src/3rd-Party,src/Test"
+      codespell -S src/3rd-Party,src/Test
 
 Useful commands::
 
-    codespell -d -q 3 --skip="*.po,*.ts,./src/3rdParty,./src/Test"
+    codespell -d -q 3 --skip="*.po,*.ts,src/3rdParty,src/Test"
 
 | List all typos found except translation files and some directories.
```

Comment:
I would argue here again that if it was `codespell -d -q 3 --skip="*.po,*.ts,src/3rdParty,src/Test"` then the note about colours and quiet level becomes mostly irrelevant, or at least you could more usefully spend the words explaining what a quiet level 3 means (especially as it involves binary flags).

Reply:
Addressed in 90cf99f